notebooks/config.yaml

```diff
@@ -1,21 +1,61 @@
-# General configurations for this project:
-# seed (int): Seed value for ensuring reproducibility in analyses that involve random operations
-#
-# root_anchor (str): A relative path to a specific file or directory used as a reference
-# (anchor) to determine the project's root directory. Example: If the project root
-# contains a .git/ folder, this can be used to dynamically locate the root directory
-# regardless of the script's location.
 general_configs:
-  seed: 0
-  root_anchor: .git/  # Default anchor pointing to the .git directory at the project root
+  # General configurations for this project
 
+  # seed (int): Seed value for ensuring reproducibility in analyses that involve random
+  # operations
+  seed: 0
 
-# Data download configurations:
-# download_chunk_size (int): The size (in KB) of data chunks to download at a time for
-# optimized performance
-#
-# github_metadata_url (str): The URL of the dataset source. In this case, the file being
-# downloaded is a zip archive, as the source contains a directory.
 download_configs:
+  # Parameters for downloading the screen data from:
+  # https://github.com/WayScience/targeted_fibrosis_drug_screen
+
+  # download_chunk_size (int): The size (in KB) of data chunks to download at a time for
+  # optimized performance
   download_chunk_size: 128
+
+  # github_metadata_url (str): The URL of the dataset source. In this case, the file being
+  # downloaded is a zip archive, as the source contains a directory.
   github_metadata_url: https://github.com/WayScience/targeted_fibrosis_drug_screen/archive/refs/heads/main.zip
```

Comment:
I am a tiny bit confused by this section, I thought the data is already downloaded? Is there a reason to include a downloading section in the config file for mAP?

Reply:
Hmm interesting! Good to know, thanks!